code/modules/research/xenoarchaeology/tools/suspension_generator.dm

```diff
@@ -361,3 +361,14 @@
 	for(var/obj/I in src)
 		I.forceMove(src.loc)
 	..()
+
+/obj/machinery/suspension_gen/AltClick(mob/user)
+	if(!user.incapacitated() && user.Adjacent(get_turf(src)))
+		if(!suspension_field)
+			if(cell.charge > 0)
+				if(anchored)
+					activate()
+				else
+					to_chat(usr, "<span class='warning'>You are unable to activate [src] until it is properly secured on the ground.</span>")
```

Comment:
Replace `usr` with `user`

Reply:
Whoops, copypasta.

I'll just update all of the usrs while I'm at it.

Reply:
:agony:
`Topic()` calls are one of the rare places where `usr` is fine because it's called directly by Byond

Reply:
*scream